backend/models/checkin.test.js

```diff
@@ -15,9 +15,9 @@ describe("Checkin Model saves the correct values", () => {
     await CheckIn.create(submittedCheckinData);
     const savedCheckinDataArray = await CheckIn.find();
     const savedCheckinData = savedCheckinDataArray[0];
-    expect(savedCheckinData.userId === submittedCheckinData.userId);
-    expect(savedCheckinData.eventId === submittedCheckinData.eventId);
-    expect(savedCheckinData.createdDate === submittedCheckinData.createdDate);
+    expect(savedCheckinData.userId).toEqual(submittedCheckinData.userId);
+    expect(savedCheckinData.eventId).toEqual(submittedCheckinData.eventId);
+    expect(savedCheckinData.createdDate.getTime()).toEqual(submittedCheckinData.createdDate);
```

Comment:
I think that you can either compare the values with toBe(), or look at comparing the object with toEqual().

Reply:
In this case they are equivalent. I prefer to use toEqual as I don't really want to check object ref. equality.

Reply:
@nickbeaird I changed to toBe